### ISSUE TITLE
Achieve higher score on "Go Report Card"

### ISSUE THREAD

**Issue by Collinux** [Collinux/gohue#2]
https://goreportcard.com/report/github.com/collinux/gohue

Issues Breakdown:
- 20% to fix on gofmt
- 10% to fix on go_vet
- 70% to fix on errcheck
- 50% to fix on golint
- 10% to fix on misspell

**Comment by Collinux** [comment]
Most issues fixed in PR @a712f307fe27070aa449452091553acf85e3d8f8

**Comment by Collinux** [comment]
```
Report for github.com/collinux/gohue
A+    Excellent!    Found 0 issues across 10 files
```
